scripts/zkai_d128_two_slice_outer_proof_object_spike_gate.py

```
@@ -0,0 +1,1009 @@
+def _validate_attempt(payload: dict[str, Any]) -> None:
+    attempt = require_object(payload.get("proof_object_attempt"), "proof object attempt")
+    for field in (
+        "outer_proof_object_claimed",
+        "pcd_accumulator_claimed",
+        "verifier_handle_claimed",
+        "two_slice_target_commitment_bound_as_public_input",
+        "selected_slice_statements_bound",
+        "selected_source_evidence_hashes_bound",
+    ):
+        if attempt.get(field) is not False:
+            raise D128TwoSliceOuterProofObjectSpikeError(f"{field} claimed without checked two-slice outer proof object")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)